src/components/big-interactive-pages/editor.tsx

```diff
@@ -66,9 +68,23 @@ let lastSavePromise = Promise.resolve();
 let saveQueueSize = 0;
 export const saveGame = debounce(
 	800,
-	(persistenceState: Signal<PersistenceState>, code: string) => {
+	(persistenceState: Signal<PersistenceState>, code: string, sessionId: string) => {
 		const doSave = async () => {
 			const attemptSaveGame = async () => {
+				const game = (persistenceState.value.kind === 'PERSISTED' && persistenceState.value.game !== 'LOADING') ? persistenceState.value.game : null;
+				if (!game) return false;
+				const lastSavedSessionInfo = localStorage.getItem('lastSavedSessionId');
```

Comment:
Please modify all usages of the 'lastSavedSessionId' string literal to a constant